decs.d.ts

```diff
@@ -137,17 +142,13 @@ declare module '@gridsuite/commons-ui' {
 
     export const RadioInput: FunctionComponent<RadioInputProps>;
 
-    export const SubmitButton: FunctionComponent<{
-        onClick: () => void;
-        disabled?: boolean;
-    }>;
+    export const SubmitButton: FunctionComponent<ButtonProps>;
 
-    export const CancelButton: FunctionComponent<{
-        onClick: (() => void) | ((event: React.MouseEvent) => void);
-        disabled?: boolean;
-        variant?: string;
-        withCustomColor?: boolean;
-    }>;
+    interface CancelButtonProps extends ButtonProps {
+        color?: string;
```

Comment:
ButtonProps has already a color prop no ?

Reply:
The prop color of ButtonProps accepts only these values: 'inherit' | 'primary' | 'secondary' | 'success' | 'error' | 'info' | 'warning'

Reply:
Like we discussed earlier, I kept that